modules/salt/pages/salt-rate-limiting.adoc

```diff
@@ -66,3 +66,32 @@ Defaults to 4 seconds.
 The presence ping gather job parameter changes the timeout setting for gathering the presence ping, in seconds.
 Adjust the [systemitem]``java.salt_presence_ping_gather_job_timeout`` parameter.
 Defaults to 1 second.
+
+
+== Disabling the Salt Mine
+
+In older versions, {productname} used a tool called Salt mine to check minion availability.
+The Salt mine would cause minions to ping the server every hour, which created significant load.
+With the introduction of a more efficient mechanism in {productname}{nbsp}3.2, the Salt mine is no longer required.
+Instead, the {productname} server uses Taskomatic to ping only the minions that appear to have been offline for twelve hours or more, with all minions being pinged at least once in every twenty four hour period by default.
+
+Newly registered Salt minions will have the Salt mine disabled by default.
+If the Salt mine is running on your system, you can reduce load by disabling it.
+This is especially effective if you have a large number of minions.
+
+Disable the Salt mine by running this command on the server:
+
+----
+salt '*' state.sls util.mgr_mine_config_clean_up
+----
+
+This will restart the minions and generate some Salt events to be processed by the server.
+However, if you have a large number of minions, handling these events could create excessive load.
```

Comment:
I would use "Thus" instead of "However" here.

Reply:
Took it out  entirely.